ci-operator/step-registry/openshift-qe/ocp-61589-max-nodes-network-expansion/openshift-qe-ocp-61589-max-nodes-network-expansion-commands.sh

```
@@ -0,0 +1,315 @@
+        fi
+    done
+    echo "All cluster operators are healthy"
+}
+
+# Function to wait for nodes to be ready
+wait_for_nodes() {
+    local target_ready=$1
+    local timeout=$2
+    echo "Waiting for $target_ready nodes to be ready (timeout: $timeout)"
+    
+    local start_time=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)